ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -106,3 +311,55 @@ func (d *nodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVo
 		},
 	}, nil
 }
+
+func recheckParameter(context map[string]string, key string, fsConfigs map[string]fileSystemConfig, fsType string) (value string, err error) {
+	v, ok := context[key]
+	if ok {
+		// This check is already performed on the controller side
+		// However, because it is potentially security-sensitive, we redo it here to be safe
```

Comment:
If they are executing the same code and nothing changed, how recheck makes it more secure?